README.md

```diff
@@ -91,33 +91,13 @@ The variable names `x` and `y` here indicate that the variables can refer to any
 
 Unfortunately, there are also a few things that are not supported at all:
```

Comment:
We can probably remove this section now. The static API avoids the confusion.

Reply:
Let's keep pointing out the difference, but we can shorten the explanation. Let's:
- drop JSBI.increment and JSBI.decrement
- the two corresponding rows from the table
- drop the "Consider this example:" part
- shorten the explanation to just:

It is impossible to replicate the behavior of the native `++` and `--` operators with static functions. Since JSBI is intended to be transpiled away eventually, it doesn't provide a similar-but-different alternative. You can use `JSBI.add()` instead.